teams/teams-ps/teams/Update-CsCustomPolicyPackage.md

```diff
@@ -0,0 +1,98 @@
+---
+external help file: Microsoft.TeamsCmdlets.PowerShell.Custom.dll-Help.xml
+Module Name: MicrosoftTeams
+title: Update-CsCustomPolicyPackage
+author: gucsun
+ms.author: gucsun
+manager: amitar
+online version: https://docs.microsoft.com/powershell/module/teams/update-CsCustomPolicyPackage
+schema: 2.0.0
+---
+
+# Update-CsCustomPolicyPackage
+
+## SYNOPSIS
+This cmdlet submits an operation that updates a custom policy package with new package settings.
+
+## SYNTAX
+
+```
+Update-CsBatchPolicyPackageAssignmentOperation -Identity <String> -PolicyList <String[]> [-Description <String>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+This cmdlet submits an operation that updates the policies into a custom policy package. It allows the user to change the description and policies included in the package. The available policy types can be found here https://docs.microsoft.com/MicrosoftTeams/manage-policy-packages.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Update-CsCustomPolicyPackage -Identity myCustomPackage -PolicyList "TeamsMeetingPolicy, Education_Teacher" , "TeamsMessagingPolicy, Firstline_Manager"
+```
+
+Updates the custom package named "myCustomPackage" by changing the policies to a new list: TeamsMeeting policy named "Education_Teacher" and TeamsMessaging policy named "Firstline_Manager" and setting description to none.
```

Comment:
phrasing: Updates the custom package named "myCustomPackage" with a new list of policies: TeamsMeeting policy named "Education_Teacher" and TeamsMessaging policy named "Firstline_Manager" and an empty description